installer/MSIX/appxmanifest.xml

```diff
@@ -48,6 +51,30 @@
             </desktop5:ItemType>
           </desktop4:FileExplorerContextMenus>
         </desktop4:Extension>
+        <uap:Extension Category="windows.fileTypeAssociation">
+           <uap3:FileTypeAssociation Name="mdpreviewhandler">
```

Comment:
Should the silent takeover flag be added here?

Reply:
Added the `flag`